client/templates/register/register.scss

```diff
@@ -1,12 +1,3 @@
-.register {
```

Comment:
this style is supposed to apply to the div on '/', should I add a register route or what?